<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetClient</class>
 <widget class="QDialog" name="DialogSetClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>131</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>设置IP：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_5">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_4">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_6">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton_10">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="pushButton_12">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Del</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pushButton_11">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_9">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSetClient</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSetClient</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
